manifests/config.pp

```diff
@@ -158,11 +158,25 @@
           content => template("rsnapshot/${script}.sh.erb"),
           mode    => '0755',
         }
-        
+
       }
     }
 
-    $cronfile = "${cron_dir}/${host}"
+    # cron on Debian seems to ignore files that have dots in their name; replace
+    # them with underscores (issue #2)
+    if $::osfamily == 'Debian' {
+      $cron_name = regsubst("${host}", '\.', '_')
```

Comment:
Hi. 

Please also add the "G" flag as my hostnames have more than one period in them! Tested in Ubuntu 16.04.

`$cron_name = regsubst("${host}", '\.', '_', 'G')`